staging/src/k8s.io/kubelet/pkg/apis/credentialprovider/OWNERS

```diff
@@ -1,10 +1,5 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
-# Disable inheritance as this is an api owners file
-options:
-  no_parent_owners: true
-approvers:
```

Comment:
I do not understand, do we need any approvers declared here?

Reply:
I moved the approves up to the parent dir `staging/src/k8s.io/kubelet/pkg/apis/OWNERS`